src/AspNetCoreModuleV2/RequestHandlerLib/stdafx.cpp

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
can we remove this stdafx.cpp file as it has no real code. We can use own precompiled header file instead VS default

Reply:
What benefit does it give us? Doesn't seem like a big deal.